### PR TITLE
[PREVIEW] Sscs 3554 name and appeal ref

### PR DESCRIPTION
**Pull request by chrisgrimble**

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sscscorbackend/controllers/QuestionController.java [hunk 1]
```diff
@@ -6,28 +6,55 @@
...
+    })
+    @RequestMapping(method = RequestMethod.GET)
+    public ResponseEntity<OnlineHearing> getOnlineHearing(
+            @ApiParam(value = "email address of the appellant", example = "foo@bar.com") @RequestParam("email") String emailAddress) {
+        try {
+            Optional<OnlineHearing> onlineHearing = onlineHearingService.getOnlineHearing(emailAddress);
+            return onlineHearing.map(ResponseEntity::ok)
+                    .orElse(ResponseEntity.notFound().build());
+        } catch (IllegalStateException exc) {
+            return ResponseEntity.unprocessableEntity().build();
```
Comment (chrisdavecm) [inline src/main/java/uk/gov/hmcts/reform/sscscorbackend/controllers/QuestionController.java]: Do we want to log these so we can see if they exist

File: src/main/java/uk/gov/hmcts/reform/sscscorbackend/service/ccd/CcdRequestDeatils.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.sscscorbackend.service.ccd;
+
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+
+@Value
+@Builder
+@EqualsAndHashCode
+public class CcdRequestDeatils {
```
Comment (chrisdavecm) [inline src/main/java/uk/gov/hmcts/reform/sscscorbackend/service/ccd/CcdRequestDeatils.java]: Spelling

File: src/main/java/uk/gov/hmcts/reform/sscscorbackend/service/ccd/domain/RegionalProcessingCenter.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
+package uk.gov.hmcts.reform.sscscorbackend.service.ccd.domain;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
```
Comment (chrisdavecm) [inline src/main/java/uk/gov/hmcts/reform/sscscorbackend/service/ccd/domain/RegionalProcessingCenter.java:9]: probably want @JsonIgnoreProperties(ignoreUnknown = true) to protect code from changes

**Review by chrisdavecm — COMMENTED** [review]
just little things

File: src/main/java/uk/gov/hmcts/reform/sscscorbackend/service/ccd/domain/RegionalProcessingCenter.java (hunk 3 above)
Comment (chrisgrimble) [reply src/main/java/uk/gov/hmcts/reform/sscscorbackend/service/ccd/domain/RegionalProcessingCenter.java:9]: This has been fixed no idea why the comment has not been marked as done

**Review by chrisdavecm — APPROVED** [review]

**Pull request merged**